MinecraftClient/Protocol/Action.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace MinecraftClient.Protocol
+{
+    public enum ActionType
```

Comment:
Both the file and enum should have the same name.
Maybe use `EntityActionType.cs` and `EntityActionType` for enum?